### DISCUSSION TITLE
Can't login to my account

### DISCUSSION THREAD

**Discussion by COOLIRON2311** [gitkraken/vscode-gitlens#3010] [Q&A]
Got  a PRO license via Student Pack. When I am about to finish authentication process, I press `Open VS Code + GitLens Pro` and nothing happens after that. Tried copy pasting authentication link with no luck.

**Comment by jkelroy** [comment]
Hello @COOLIRON2311, I see you have also submitted a support ticket and I have followed up via email to troubleshoot this further.
